src/components/users/forgotPassword/ResetPassword.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/require-default-props */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import Input from '../../common/Input';
+import Button from '../../common/Button';
+import { resetPassword } from '../../../actions/resetPassword';
+import { setAlert } from '../../../actions/alert';
+import Alert from '../../layouts/Alert';
+
+export class ResetPassword extends Component {
+  constructor() {
+    super();
+    this.state = {
+      password: '',
+      password2: ''
+    };
+
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  onChange(e) {
```

Comment:
Change these to arrow functions so that you do not bind them. Also, you will have to lose the constructor and declare the state directly.